src/common/telemetry.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
```

Comment:
Do we have `axios` as dependency in package.json ?

Reply:
I have updated the `package.json`, I need something simple to send a HTTP request so `axios` is not a must here. If you know any libraries that can do the same job in existing dependencies, happy to remove `axios` too.

Reply:
I guess we can stick with this for now?